### PR TITLE
Fix logo vectorization and update course view

### PR DESCRIPTION
**Pull request by ballyalley-o**
This pull request updates the logo to a vector format, making it the official logo for the project. The logo file has been replaced with a vector version, and the necessary changes have been made in the code to use the new logo. This improves the scalability and quality of the logo.

### CODE REVIEW

File: src/config/asset-directory.ts [hunk 1]
```diff
@@ -21,6 +21,7 @@ const _svg = (svg: string) => {
 const ASSET = {
   // bg
   _BG_TEMP: 'https://images.pexels.com/photos/9672971/pexels-photo-9672971.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=2',
```
Comment (in-a-iii) [inline src/config/asset-directory.ts:23]: move this to .env

**Review by in-a-iii — CHANGES_REQUESTED** [review]

File: src/config/asset-directory.ts (hunk 1 above)
Comment (ballyalley-o) [reply src/config/asset-directory.ts:23]: > move this to .env

yes sure.

File: src/config/asset-directory.ts [hunk 2]
```diff
@@ -20,8 +22,7 @@ const _svg = (svg: string) => {
 
 const ASSET = {
   // bg
-  _BG_TEMP: 'https://images.pexels.com/photos/9672971/pexels-photo-9672971.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=2',
-  default_badge: 'https://thecodecoachprojct.com/upload/badge/no-badge.png',
```
Comment (in-a-iii) [inline src/config/asset-directory.ts]: Good to see you removed the unused bg temp

File: src/config/global.ts [hunk 3]
```diff
@@ -4,6 +4,7 @@ const GLOBAL = {
   APP_SERVER: import.meta.env.VITE_APP_SERVER,
   APP_ORIGIN: import.meta.env.VITE_APP_ORIGIN,
   APP_ENV: import.meta.env.VITE_APP_ENV,
+  DEFAULT_BADGE: import.meta.env.VITE_DEFAULT_BADGE,
```
Comment (in-a-iii) [inline src/config/global.ts:7]: Great, good to go for me,

**Review by in-a-iii — APPROVED** [review]

**Pull request merged**